components/Footer.js

```diff
@@ -0,0 +1,27 @@
+const Footer = `
```

Comment:
Make your module a function. See the course I shared for how and why.

Reply:
Do this for all your modules.

Reply:
Okay. I will work towards that.